auto-pos-creator/tests/create_pair_pos_test.rs

```diff
@@ -533,7 +533,7 @@ fn enter_lp_farm_and_metastaking_through_pos_creator_test() {
         .assert_ok();
 
     let expected_remaining_second_token = 3_780_718u64;
-    let expected_dual_yield_token = 41_666_665u64;
+    let expected_dual_yield_token = 43_478_260u64;
```

Comment:
Why are these values different?

Reply:
Because it references the updated code from the Pair SC, where Safe Price is computed a bit different and returns different data. The only changes should come from functions that query the safe price.